packages/components/cascader-panel/src/index.vue

```diff
@@ -268,7 +268,7 @@ export default defineComponent({
       if (props.props.multiple) {
         reactive(newNodes).forEach((node) => node.doCheck(true))
       } else {
-        newNodes.forEach((node) => node.doCheck(true))
+        reactive(newNodes).forEach((node) => node.doCheck(true))
```

Comment:
Lines 269 and 271 can be combined into one line.

Reply:
Done, please take a look.